### PR TITLE
Feature/insert sample text

### PR DESCRIPTION
**Pull request by bkrmendy**
## Description
This PR enables inserting sample text in a single step from the `Add element` menu and the insert menu. Currently, this can only be done by selecting `span` and checking `Add content`.

## Details
- a new insertable group is added (`SAMPLE_ELEMENT`) with a single entry (`Sample text`)
- a new test is added to the test suite corresponding to the floating insert menu

## Limitations
- no ChatGPT support

### CODE REVIEW

File: editor/src/components/shared/project-components.ts [hunk 1]
```diff
@@ -366,6 +375,19 @@ const fragmentElementsDescriptors: ComponentDescriptorsForFile = {
   },
 }
 
+const samplesDescriptors: ComponentDescriptorsForFile = {
+  sampleText: {
+    properties: {},
+    variants: [
+      {
+        insertMenuLabel: 'Sample text',
+        elementToInsert: jsxElementWithoutUID('span', [], [jsxTextBlock('Utopia rules')]),
```
Comment (balazsbajorics) [inline editor/src/components/shared/project-components.ts:384]: `'Utopia rules'` :D ahem maybe we want to workshop the inserted string, cc @maltenuhn

Comment (bkrmendy) [reply editor/src/components/shared/project-components.ts]: updated to `Sample text` in https://github.com/concrete-utopia/utopia/pull/3550/commits/a92586564f411a37e4635db01efba3120e122db9

File: editor/src/components/shared/project-components.ts [hunk 2]
```diff
@@ -77,6 +78,10 @@ export function clearInsertableComponentUniqueIDs(
   }
 }
 
+export interface InsertableComponentSamples {
```
Comment (ruggi) [inline editor/src/components/shared/project-components.ts:81]: shall this be renamed to `InsertableComponentGroupSamples` to be consistent with the other siblings?

**Review by ruggi — APPROVED** [review]

File: editor/src/components/shared/project-components.ts (hunk 2 above)
Comment (bkrmendy) [reply editor/src/components/shared/project-components.ts]: great idea! renamed it in https://github.com/concrete-utopia/utopia/pull/3550/commits/7021566fc9ef1b8c7c538b2b61eeb1a051811451

**Review by enidemi — APPROVED** [review]

**Pull request merged**